CONTRIBUTING.md

```diff
@@ -62,7 +62,16 @@ included in the project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
+4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
+
+    ```bash
+    # Install the needed gems
+    gem install jekyll jekyll-sitemap pygments.rb
```

Comment:
Better yet: `gem install github-pages` will install all gems which are available and potentially needed for a GH Pages build, including setting them to the correct versions for GH Pages.